src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -732,6 +732,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <NuGetLockFilePath>$(NuGetLockFilePath)</NuGetLockFilePath>
         <RestoreLockedMode>$(RestoreLockedMode)</RestoreLockedMode>
         <_CentralPackageVersionsEnabled>$(_CentralPackageVersionsEnabled)</_CentralPackageVersionsEnabled>
+        <CentralPackageVersionOverrideEnabled>$(EnablePackageVersionOverride)</CentralPackageVersionOverrideEnabled>
```

Comment:
Usually the configuration properties are named starting with `Restore`. 
Do you think that should be the case here as well?

Reply:
This is a great question, I wasn't sure about the name of this property either.  I did notice that some of the properties are prefixed with `Restore` and I think that makes sense if they directly tweak settings for restore.  In this case, its more of a "feature of the restore graph" or whatever and not really a knob that controls restore itself.

For example, the property to disable/enable central package management is `ManagePackageVersionsCentrally`.  The property to tell restore if interactivity is enabled/disabled is `NuGetInteractive`.  So I wasn't sure what to name this thing.

 I don't feel strongly one way or the other.  Here are some potential names:
* `EnablePackageVersionOverride`
* `AllowPackageVersionOverride`
* `RestoreEnablePackageVersionOverride`
* `RestoreAllowPackageVersionOverride`

What does everyone think?

Reply:
Honestly, I'm comfortable with either of the 4 :D 

I like the `RestoreEnablePackageVersionOverride`, or `EnablePackageVersionOverride`, even if `Allow` is technically more correct.

@zivkan thoughts?

Reply:
I don't know. Since MSBuild properties could be for anything, NuGet, or the compiler, or output directories, or other things, I see value in a naming convention. On the other hand, many features like implicit usings, just have a minimal name, assuming the customer understands the context for which it's being used, so `ManagedPackageVersionsCentrally` and `CentralPackageVersionOverrideEnabled` seems to fit.

Perhaps if you want another opinion, try asking people on the .NET SDK team.

Reply:
I actually like CentralPackageVersionOverrideEnabled, because it's more explicit, but the current proposal is EnablePackageVersionOverride.

Reply:
Okay I'm going to change the name to `CentralPackageVersionOverrideEnabled`